src/tests/unit/tests/reports/components/report-sections/report-body.test.tsx

```diff
@@ -26,11 +27,11 @@ describe('ReportBody', () => {
             pageUrl,
             description: 'test description',
             scanDate: new Date('2019-05-29T19:12:16.804Z'),
-            environmentInfo: {
-                axeCoreVersion: 'axe-core-version',
-                browserSpec: 'browser-spec',
-                extensionVersion: 'extension-version',
-            },
+            environmentInfoProvider: new EnvironmentInfoProvider(
```

Comment:
Same here as in the other test: this should be a mock, not an instance of the actual class 'FooterText' has a dependency on.